implementation/exporters/src/main/java/io/smallrye/opentelemetry/implementation/exporters/AbstractVertxExporterProvider.java

```diff
@@ -32,6 +38,10 @@ public abstract class AbstractVertxExporterProvider<T extends Marshaler> {
 
     private static final String MIMETYPE_PROTOBUF = "application/x-protobuf";
 
+    private static final String OTEL_EXPORTER_VERTX_CDI_QUALIFIER = "otel.exporter.vertx.cdi.identifier";
```

Comment:
One more thing: Since this is a SmallRye feature and not Otel, should this sysprop reflect that? Say, `smallrye.otel.exporter.vertx.cdi.identifier` or am I overthinking it?

Reply:
I am not sure, but would it break https://github.com/wildfly/wildfly/blob/34.0.0.Beta1/microprofile/telemetry-smallrye/cdi-provider/src/main/java/org/wildfly/extension/microprofile/telemetry/api/MicroProfileTelemetryCdiExtension.java#L39 if it starts with `smallrye` ?

Reply:
Yeah, that's fair. Ignore this one. :)